resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/repository/ReactiveRepositoryRedisAdapter.java

```diff
@@ -0,0 +1,371 @@
+/*
+ * Copyright 2021 Crown Copyright
```

Comment:
As per PAL-569, copyright should be a range of years 2018-2021, update in your template, but we can leave this here for now until we work on that story

Reply:
I'm going to leave this for now so that it can be updated as part of PAL-569.